lua/entities/gmod_wire_expression2/core/wirelink.lua

```diff
@@ -28,7 +30,6 @@ end
 
 local function validWirelink(self, ent)
 	if not IsValid(ent) then return false end
-	if not ent.extended then return false end
 	if not isOwner(self, ent) then return false end
 	return true
 end
```

Comment:
This is the cause of #2940. It's not a backwards compatibility problem, it was an oversight. There's no purpose for entities to store this flag by default; there is no unique restrictions for wirelinks. Wirelinks are just entities.

If you can give me a good reason why someone would use the wirelink output versus a wirelink variable, I'll concede.

Reply:
To be honest `extended` isn't even a good name for marking an wirelink. A field `wirelinkExist` or something would have been better. Changing it however could break BC to addons.

Reply:
> To be honest `extended` isn't even a good name for marking a wirelink. A field `wirelinkExist` or something would have been better. Changing it however could break BC to addons.

`hasWirelink` would be best imo